src/python/bot/fuzzers/libFuzzer/strategy_selection.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Generate strategy pool for launcher based on strategy probabilities.
+
+Decides the set strategies to be considered by the launcher. Note
+that because of compatability issues, the exact set of strategies
+generated here may be modified in the launcher before being launched."""
+
+from bot.fuzzers import engine_common
+from bot.fuzzers import strategy
+
+# Probability of using `-max_len` option. Not applicable if already declared in
+# .options file.
+RANDOM_MAX_LENGTH_PROBABILITY = 0.15
+
+# Probability of doing ML RNN mutations on the corpus in this run.
```

Comment:
@mbarbella-chromium 
Do you think these comments are worth keeping? They don't seem useful to me (except in cases where they do more than explain what's clear, like on line 23.

Reply:
Nope. Definitely seems like it would be better to remove them and put all of the _PROBABILITY constants in one block.